website/docs/reference/resource-configs/tags.md

```diff
@@ -164,10 +200,65 @@ seeds:
 
 </File>
 
+### Apply tags to saved queries
+
+<VersionBlock lastVersion="1.8">
+
+Applying tags to saved queries is only available in dbt Core versions 1.9 and later.
```

Comment:
Hiya @mirnawong1

This is non-blocking suggestion. Would this benefit from an info callout? It may encourage the user to upgrade?

Kind Regards
Natalie

Reply:
good shout, will add!